src/doc/book/structs.md

```diff
@@ -163,11 +163,48 @@ struct Point(i32, i32, i32);
 let black = Color(0, 0, 0);
 let origin = Point(0, 0, 0);
 ```
+
 Here, `black` and `origin` are not equal, even though they contain the same
 values.
```

Comment:
This sentence could be confusing even though I get what it is trying to say. `Color(0, 0, 0)` and `Color(0, 0, 0)` are also not "equal" in some sense because we didn't derive `PartialEq`.